api/services/auth.services.ts

```diff
@@ -1,13 +1,12 @@
-const User = require("../models/Users.models");
+import { User } from "../models/Users.models";
+import { User as UserType } from "../types/User";
 
-class AuthServices {
+export class AuthServices {
   static async getUserbyEmail(email: string) {
     return await User.findOne({ email: email }).exec();
   }
 
-  static async createUser(data: string) {
+  static async createUser(data: UserType) {
```

Comment:
This should come from `../types/User` (when we have `index.ts`, from `../types`).
It should be named: `UserCreation`